web/docs/Frontend Starters/Next.js/Next.js + WordPress/styling-configuration.md

```diff
@@ -0,0 +1,83 @@
+---
+id: 'styling-configuration'
+title: 'Styling Configuration'
+sidebar_position: 1
+slug:
+  '/Frontend Starters/Next.js/Next.js + WordPress/Styling configuration'
+---
+
+## Before You Begin
+
+This guide assumes the reader has a little to no experience with Tailwind CSS. If you are new to Tailwind CSS, we recommend reading the [Tailwind CSS documentation](https://tailwindcss.com/docs) before proceeding.
+
+## Block Editor
+
+The block editor plugin for Tailwind enables to render the content from your WordPress site to the Next.js frontend, with parallel styling and some opinionated changes. To use this plugin you must have the WordPress-Kit installed and configure the `tailwind.config.js` file.
```

Comment:
> To use this plugin you must have the WordPress-Kit installed

Maybe put this part in the Before You Begin section? Also let's wrap it in backticks like so: `wordpress-kit`